pt-BR/mail/chrome/messenger/shutdownWindow.properties

```diff
@@ -37,7 +37,7 @@
 
 
 # These strings are loaded and represented by the XUL dialog.
-shutdownDialogTitle=Shutdown Progress Window
-taskProgress=Processing %1$S of %2$S Tasks 
+shutdownDialogTitle=Janela de Progresso de Fechamento
```

Comment:
CORRECTIONS NEEDED it should be shutdownDialogTitle=Janela de Progresso de Encerramento